### PR TITLE
Update .travis.yml

### PR DESCRIPTION
**Pull request by ecadius**
The builds should run now. refs: #8639 #8618 #8617 #8245

### CODE REVIEW

**Comment by ecadius** [comment]
What has time changed that they don't run anymore? I don't get it. 3 Months ago the build passed: https://travis-ci.com/ecadius/jquery-mobile/builds/99162130

File: .travis.yml [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
   directories:
   - node_modules
 
 before_script:
   - npm install -g grunt-cli
   - npm install
+  - npm install -g casperjs
 
-script: npm run-script ci
+script: npm run-script test
```
Comment (apsdehal) [inline .travis.yml]: I believe we don't need to change this.

File: .travis.yml [hunk 2]
```diff
@@ -1,21 +1,21 @@
 language: php
 
 php:
-  - "5.5"
+  - "7.0"
```
Comment (apsdehal) [inline .travis.yml]: IIRC, @arschmitz used to say that there were some issues in upgrading to PHP 7.0.

**Comment by apsdehal** [comment]
Can you retrigger the build by pushing a commit?

**Comment by ecadius** [comment]
I reseted the php version back to 5.5. Perhaps the travis tests will pass, but they are a bit random and i simple try to rebuild my local environment which passed all tests.

File: .travis.yml (hunk 1 above)
Comment (ecadius) [reply .travis.yml]: In the ReadMe describes the test process with `npm run-script test`. What is the purpose of `npm run-script ci`?

**Comment by apsdehal** [comment]
Only one of the tests' suites is failing, we can figure that one later (has some issue related to phatomjs). Squashing and merging this one for now. Thanks.

**Comment by mgol** [comment]
@apsdehal The issue at https://travis-ci.org/jquery/jquery-mobile/jobs/545864736 is not related to PhantomJS in particular but to the usage of `jQuery.isArray` which has been removed on jQuery `master` and will not be in jQuery 4.0.

**Pull request merged**